package-template.sile-dev-1.rockspec

```diff
@@ -0,0 +1,28 @@
+package = "package-template.sile"
+version = "dev-1"
+
+description = {
+  summary = "A template for SILE packages...",
+  detailed = [[]],
+  homepage = "", -- e.g. its GitHub repo
+  maintainer = "",
+  license = "MIT"
+}
+
+source = {
+  url = "", -- a clonable repository link
```

Comment:
I took me a while to understand why some of my GitHub packages didn't work in some cases, until someone told me I had to change my initial `git:` prefix to |`git+https:` -- And it did the trick. So perhaps we could help the users here, with a working example (or a useful comment)